inflection/src/inflection/grammar/synthesis/KoGrammarSynthesizer_ParticleResolver.cpp

```diff
@@ -76,7 +75,7 @@ std::u16string KoGrammarSynthesizer_ParticleResolver::getRelevantString(std::u16
     }
     if (!strToTokenize.empty() && !inflection::lang::StringFilterUtil::HANGUL_SCRIPT().contains((UChar32)inflection::util::StringViewUtils::codePointAt(strToTokenize, int32_t(strToTokenize.length() - 1)))) {
         // We don't care about the strings in the phonetic Hangul script. Return the relevant last word to check without punctuation.
-        std::unique_ptr<inflection::tokenizer::TokenChain> tokenChain(npc(npc(englishTokenizer)->createTokenChain(std::u16string(strToTokenize))));
+        std::unique_ptr<inflection::tokenizer::TokenChain> tokenChain(npc(englishTokenizer->createTokenChain(std::u16string(strToTokenize))));
```

Comment:
Please ensure that englishTokenizer gets an npc check during construction, since you're removing it from here.

Reply:
PTAL